some typo fixes, more fluid readability and fixes of weird machine translated terms for the english parts of the guide (#10231)
MaaAssistantArknights/MaaAssistantArknights: docs/en-us/protocol/integrated-strategy-schema.md

    In addition to those general groups, sometimes we need more custom tweaks for some operators or types such as
    | Group | Operators | Features |
    | :--- | :--- | :--- |
    | Thorns | Thorns, Horn | Long-range melee, many maps have good spots |
    | Summoners | Kal'tsit, Ling, Scene | Melee blocking, some maps need deployment prioritization, summons can be used as blocks or fodder |

>     | Agent | Cantabile, Ines | Has DP recovery, deals DPS and its single block |

Reviewer: ```suggestion
    | Agent | Cantabile, Ines | Has DP recovery, deals DPS and is single block |
```

    | Skadi Alter | Skadi the Corrupting Heart | Low DP cost, special range, some maps have optimal spots |
    | Reed Alter | Reed the Flame Shadow | In Sami I.S. she's commonly used as an opening caster, DPS and healing, some maps have optimal spots |
    | SilverAsh | SilverAsh, Młynar | Ground operators with large range, very useful against bosses |
    | Surtr | Surtr | Since Surtr always carries the 3rd skill, her positioning ability is almost zero and deployment priority is extremely low |

>     | Dice | Dice | In Mizuki I.S. the dice need to be operated separately |

Reviewer: ```suggestion
    | Dice | Dice | In Mizuki I.S. the dice needs to be operated separately |
```
This is 3rd person.

### Basic I.S. fighting logic of MAA

> 1. Perform basic combat operations based on the tileset of the map

Reviewer: ```suggestion
1. Perform basic combat operations based on the tile grid of the map
```
Tileset is related to game art design (Like Minecraft particles)
